ntoskrnl/mm/i386/page.c

```diff
@@ -442,12 +409,11 @@ MmIsDisabledPage(PEPROCESS Process, PVOID Address)
         ASSERT(Process != NULL);
         ASSERT(Process == PsGetCurrentProcess());
 
-        MiLockProcessWorkingSetShared(Process, PsGetCurrentThread());
+        NT_ASSERT(PsGetCurrentThread()->OwnsProcessWorkingSetExclusive || PsGetCurrentThread()->OwnsProcessWorkingSetShared);
```

Comment:
```suggestion
        NT_ASSERT(PsGetCurrentThread()->OwnsProcessWorkingSetExclusive ||
                  PsGetCurrentThread()->OwnsProcessWorkingSetShared);
```
Why using an NT_ASSERT here, but in other places where similar check is done, a simple ASSERT was used instead?

Reply:
NT_ASSERT uses annotations, while ASSERT is just a macro forwarding to RtlAssert

Reply:
We define ASSERT to NT_ASSERT, so no need to rewrite all ASSERTs 😉